cmd/wallet-web/src/store/modules/user.js

```diff
@@ -150,8 +151,8 @@ export default {
         ? {
             username: state.username,
             setupStatus: state.setupStatus,
-            profile: state.profile,
-            preference: state.preference,
+            profile: toRaw(state.profile),
```

Comment:
getCurrentUser() gets called very frequently, is there any way to optimize it to avoid calling `raw()` everytime?